scraper/src/libretexts2zim/third_party/README.md

```diff
@@ -0,0 +1 @@
+This folder contains resources fetched from third parties, with accompaying license information.
```

Comment:
@kelson42 do you have any feedback? Here we duplicate the LibreText icon to avoid breaking the scraper should the icon move. It is used as default illustration for the ZIM.

I've removed the big icon since it is not used in the end.

Reply:
Self-resolve this if he doesn't answer by next round of review. Can always be reverted

Reply:
Closing since here is the new round ^^